src/python/bot/startup/heartbeat.py

```diff
@@ -25,6 +25,8 @@
 import sys
 import time
 
+from health_responser import run_server as run_health_check_responser_server
```

Comment:
nit: just `run_health_check_server` is shorter and just as descriptive.

Reply:
Changed to `run_health_responder_server` as I presume we are the responder to the health checker?